app/controllers/cart_controller.rb

```diff
@@ -6,6 +6,21 @@ def add_item
     redirect_to "/items"
   end
 
+  def add_quantity
+      item = Item.find(params[:item_id])
+      quantity = params[:quantity].to_i
+      inventory = item.inventory
+      cart.add_quantity(quantity, item, inventory)
+      redirect_to '/cart'
+  end
+
+  def decrease_quantity
+    item = Item.find(params[:item_id])
+    quantity = params[:quantity_decrease].to_i
+    cart.remove_quantity(quantity, item)
+    redirect_to '/cart'
+  end
```

Comment:
These are very repetitive and I can definitely clean it up. I could use if statements to determine if the user is adding or subtracting from the quantity. A good refactor opportunity. But for now, it works.